.circleci/test-deploy.yml

```diff
@@ -0,0 +1,49 @@
+version: 2.1
+
+orbs:
+  maven: circleci/maven@dev:<<pipeline.git.revision>>
+  orb-tools: circleci/orb-tools@11.1
+
+filters: &filters
+  tags:
+    only: /.*/
+
+jobs:
+  # This job is an example of an integration testing job.
+  # This job should execute a command from your orb and verify
+  # the output is as expected, otherwise the job should fail.
+  #
+  # Rename this job and add more as needed.
+  #
+  integration-test-1:
```

Comment:
I believe this test serves mostly to test the docker image, and is not needed.

Reply:
Right. Good point, I'll make it actually run test on the sample app